api-docs/get-interactions-hours.md

```diff
@@ -0,0 +1,60 @@
+# Number of Interactions per Hour Range.
+
+Returns the total number of clients who have had at least that many client interaction hours in the given time period, along with the list of clients that match the specified criteria.
+
+## Request
+
+_The request should accept query parameters indicating a time range, number of client interaction hours, and current page._
+
+```http
+GET /api/interactions?start=2019-05-06T0&end=2019-05-08T0&=21600&hours=21600&page=1
```

Comment:
Lets put all reports under `/api/reports`. And maybe try to put more into the URL's name so that it's easy to understand why it's returning a list of clients instead of a list of interactions.

```suggestion
GET /api/reports/interaction-hours-by-client?start=2019-05-06T0&end=2019-05-08T0&minInteractionHours=21600&page=1
```

Reply:
Is the `T0` at the end of the string to indicate timezone? If so, that's a nice little touch so that we aren't just assuming UTC in the db query when we likely want MST.

Reply:
Yep! It could help to query time periods more precisely.